### PR TITLE
Docker instructions improvements

### LINKED ISSUE #17: Docker usage instructions link missing
The link to the instructions for how to run tinygo with Docker from the installation page is 404ing:

https://tinygo.org/getting-started/linux/
(see bottom of page).

We should update the docs with the new docker image in mind. Here's a oneliner I've successfully used to build and copy the relevant wasm files that can be used for inspiration:

```
docker run --rm -v $$(pwd):/go/src/github.com/johanbrandhorst/wasm-experiments tinygo/tinygo:0.5.0 /bin/bash -c "\
	cd /go/src/github.com/johanbrandhorst/wasm-experiments && \
	tinygo build -o ./html/test.wasm -target wasm --no-debug ./$(target)/main.go && \
	cp /usr/local/tinygo/targets/wasm_exec.js ./html/wasm_exec.js\
"
```
(From https://github.com/johanbrandhorst/wasm-experiments/blob/6d867a75ab1291a26cc1af5ae2655d82e9f88af3/Makefile#L8).

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Fixes #17

### CODE REVIEW

File: content/getting-started/using-docker.md [hunk 1]
```diff
@@ -23,9 +29,11 @@ To compile `blinky1.hex` targeting an AVR microcontroller such as the Arduino:
 
     docker run --rm -v $(pwd):/src tinygo/tinygo tinygo build -o /src/blinky1.hex -size=short -target=arduino examples/blinky1
 
-For projects that have remote dependencies outside of the standard library and go code within your own project, you will need to map your entire GOPATH into the docker image in order for those dependencies to be found:
+For projects that have remote dependencies outside of the standard library and
```
Comment (justinclift) [inline content/getting-started/using-docker.md]: Maybe "with remote dependencies" ?

Comment (justinclift) [reply content/getting-started/using-docker.md]: Actually, maybe not... the "within" below makes that not sound quite right. :wink:

File: content/getting-started/using-docker.md [hunk 2]
```diff
@@ -23,9 +29,11 @@ To compile `blinky1.hex` targeting an AVR microcontroller such as the Arduino:
...
+go code within your own project, you will need to map your entire `$GOPATH`
+into the docker image in order for those dependencies to be found:
```
Comment (justinclift) [inline content/getting-started/using-docker.md]: The "in order" can probably just be dropped.

**Comment by deadprogram** [comment]
Very good clarifications thanks @johanbrandhorst and @justinclift for working on this. Now merging.

**Pull request merged**